src/Cake/Cake.Tool.csproj

```diff
@@ -9,6 +9,17 @@
 
   <PropertyGroup>
     <PackageId>Cake.Tool</PackageId>
+    <Authors>Patrik Svensson; Mattias Karlsso; Gary Ewan Park; Alistair Chapman; Martin Björkström; Dave Glick; Pascal Berger</Authors>
```

Comment:
Shared elements imported for all projects shouldn't be specified again. 
For reference see: https://github.com/cake-build/cake/blob/develop/src/Shared.msbuild

Reply:
Sorry didn't know about the Shared.msbuild. I changed it.